### PR TITLE
docs: added tencent cloud user case

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

added tencent cloud user case

### CODE REVIEW

File: website/blog/2021-09-16-tencent-cloud.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+### 为什么选择了 Apache APISIX？
+
+Apache APISIX 在存储依赖和路由规则这两方面做的非常好，很适合我们的业务场景。我们的业务比较看重路由灵活度和路由匹配算法。目前已接入 50 多个上游和数百条路由规则，所以我们希望路由匹配的性能越高越好。Apache APISIX 路由匹配算法复杂度明显优于 Kong，且配置生效时间小于 1ms，单核 QPS 远高于 Kong。综合以上技术和操作层面，我们最终选择了 Apache APISIX。
+
+## 基于 Apache APISIX 的架构调整
+
+接入 Apache APISIX 后，我们完成了智能钛机器学习平台网关方面的功能，解决了之前关于技术和业务层面的需求。
+
+从这张图可以看到， Apache APISIX 支持 http+pb、http+json、gRPC、WebSocket等流量。这些流量经过了 Apache APISIX 之后，会走向智能钛机器学习平台定制开发的一些组件。
```
Comment (SylviaBABY) [inline website/blog/2021-09-16-tencent-cloud.md]: WebSocket 等流量

File: website/blog/2021-09-16-tencent-cloud.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+## 调研对比
+
+进入我们考察名单的有：Envoy、Kong 以及 Apache APISIX。我们从多维度对上述三个产品进行了对比，结果如下。
+
+![竞品分析与对比](https://static.apiseven.com/202108/1631781764958-a2cadf83-80b4-4b50-ba42-76b21d0d211a.jpg)
+
+因为 Envoy 的技术栈是 C++，需要定位问题时，很可能要去看 C++ 的源代码。这种定位问题可能会对我们造成一定的影响，所以 Envoy 这个方案否决得比较早。
+
+Kong 和 Apache APISIX 所使用的技术栈相同，都是 OpenResty。但是在存储依赖这一栏，Kong 依赖的是关系型数据库 PostgreSQL。在软件行业，数据库的高可用配置是非常复杂的。不仅需要配备专门的DBA，而且实施难度也非常大。关系型数据库太重了，我们并不需要使用关系型数据库来保证 ACID 和 MVCC。
```
Comment (SylviaBABY) [inline website/blog/2021-09-16-tencent-cloud.md]: 专门的 DBA

File: website/blog/2021-09-16-tencent-cloud.md (hunk 1 above)
Comment (yzeng25) [reply website/blog/2021-09-16-tencent-cloud.md]: fixed

File: website/blog/2021-09-16-tencent-cloud.md (hunk 2 above)
Comment (yzeng25) [reply website/blog/2021-09-16-tencent-cloud.md]: fixed

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by liuxiran — APPROVED** [review]

**Pull request merged**